src/styles/card.scss

```diff
@@ -0,0 +1,71 @@
+.card {
+  width: 200px;
+  border: 1px solid white;
+  box-sizing: border-box;
+  background-color: #fff;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  &__title {
+    margin: 40px 0 0;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: #060b35;
+  }
+
+  &__sudtitle {
+    margin: 4px 0 0;
+    padding-top: 0;
+    font-weight: 400;
+    line-height: 14px;
+    color: #616070;
+    font-size: 10px;
+  }
+
+  &__price-container {
+    margin-top: 24px;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+  }
+
+  &__price-label {
+    margin: 0;
+    color: #616070;
+    font-size: 12px;
+    line-height: 18px;
+  }
+
+  &__money {
+    margin: 0;
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+    color: #060b35;
+  }
+
+  &__buy-button {
+    margin: 16px 0 0;
+    background-color: #00acdc;
+    position: relative;
+    width: 166px;
+    height: 40px;
+    border-radius: 5px;
+    color: #fff;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    text-align: center;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+    text-decoration: none;
+    box-sizing: border-box;
+    &:hover {
+      background: #fff;
+      color: #00acdc;
+      border: 1px solid #00acdc;
```

Comment:
Ensure that the hover state for the `&__buy-button` is consistent with the design requirements. If the design specifies different colors or styles, they should be implemented accordingly. Additionally, when changing `background-color` on hover, it's better to transition the color change for a smoother visual effect. Consider adding `transition: background-color 0.3s;` to the `&__buy-button` block.